logic/processor.py

```diff
@@ -33,6 +39,122 @@ def _coffee(text):
 def _translate(text):
     return Response(text="")
 
+    
+def temperature_conversion(temp, logging):
+    '''
+    #Author: Nils Becker
+    Converts a given temperature in °K into a °C String representation
+    :param temp: Float or int Value of temperature in °K
+    :param logging: Currently used instance of logging module
+    :returns: The temperature converted to °C as a String
+    '''
+    try:
+        return f"{int(temp)-273}°C"
+    except ValueError:
+        logging.error(f"{temp} must be an int or float")
+        pass
```

Comment:
the method returns nothing and doesn't raise an error if problem is in format. Thus it returns a "silent" None. Not good.